### PR TITLE
incr eos connector timetout

### PR DESCRIPTION
**Pull request by ericdong66**
eos swi is getting bigger, incr the timeout value to avoid socket timeout.

### CODE REVIEW

File: pylib/aeon/eos/connector.py [hunk 1]
```diff
@@ -71,7 +71,8 @@ def configure(self, contents):
         if not isinstance(contents, list):
             raise RuntimeError('contents must be a list, for now')
 
-        contents = ['enable', 'configure'] + filter(bool, contents)   # filters out empty lines
+        # filters out empty lines
+        contents = ['enable', 'configure'] + list(filter(bool, contents))
```
Comment (hk-spark) [inline pylib/aeon/eos/connector.py]: Filter always returns a list right, why make it a list again ?

File: tox.ini [hunk 2]
```diff
@@ -41,4 +41,5 @@ commands = flake8 pylib
 # Ignore the following pep8 violations
 # E501: 80 character line length limit
 # E128: continuation line under-indented for visual indent
-ignore = E501, E128
\ No newline at end of file
+# W605: invalid escape sequence
+ignore = E501, E128, W605
```
Comment (Ragsboss) [inline tox.ini]: What's the reason for excluding this? Can we instead fix the offending lines or do a local ignore if fixing is not feasible..

File: pylib/aeon/eos/connector.py (hunk 1 above)
Comment (ericdong66) [reply pylib/aeon/eos/connector.py]: can be a filter object depend on python version

File: tox.ini (hunk 2 above)
Comment (ericdong66) [reply tox.ini]: problem is ci env tools moved on, there is no  offending lines here.

**Pull request merged**